packages/react-theme-provider/src/index.ts

```diff
@@ -1,23 +1 @@
-import './version';
-
-export * from './ThemeProvider';
-export * from './ThemeProvider.types';
-export * from './useThemeProviderClasses';
-export * from './useThemeProvider';
-export * from './useThemeProviderState';
-export * from './withThemeProvider';
-export * from './applyClasses';
-export { useTheme } from './useTheme';
-export { ThemeContext } from './ThemeContext';
-export * from './types';
-export * from './getStyleFromPropsAndOptions';
-export * from './tokensToStyleObject';
-export * from './useInlineTokens';
-export * from './makeVariantClasses';
-export * from './makeStyles';
-export * from './makeClasses';
-export { IStyle, IRawStyle, IRawFontStyle, IStyleFunctionOrObject } from '@fluentui/merge-styles';
-
-export * from './styleRenderers/mergeStylesRenderer';
-export * from './styleRenderers/types';
-export * from './styleRenderers/useStyleRenderer';
+export const noop = 'noop';
```

Comment:
We cannot simply do not have exports, `api-extractor` fails 🦉

Reply:
can you double check if you still need to add this after adding `import './version'`?

Reply:
Nope